docs/releases.md

```diff
@@ -0,0 +1,60 @@
+## Checklist
+
+Helix releases are versioned in the Calendar Versioning scheme:
+`YY.0M(.MICRO)`, for example `22.05` for May of 2022. In these instructions
+we'll use `<tag>` as a placeholder for the tag being published.
+
+* Merge the changelog PR
+* Edit the `VERSION` file and remove the `-dev`
+* Tag and push
+    * `git tag -s -m "<tag>" -a <tag> && git push`
+    * Make sure to switch to master and pull first
+* Edit the `VERSION` file and add back the `-dev` and change the date to the next planned release
+    * Releases are planned to happen every two months, so `22.05` should be changed to `22.07-dev` in this step
```

Comment:
Let's switch back to bumping the version on release. It makes the workflow a bit simpler, we'll just need to be more careful when reading github issues to distinguish dev vs stable. Perhaps we can add that as a field to the debug template?